melodic/distribution.yaml

```diff
@@ -6325,6 +6325,23 @@ repositories:
       url: https://github.com/OUXT-Polaris/rostate_machine.git
       version: master
     status: developed
+  rosthrottle-release:
+    doc:
+      type: git
+      url: https://github.com/UTNuclearRoboticsPublic/rosthrottle-release.git
+      version: 1.2.0
+    release:
+      packages:
+      - rosthrottle
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/UTNuclearRoboticsPublic/rosthrottle-release.git
+      version: 1.2.0-2
+    source:
+      type: git
+      url: https://github.com/UTNuclearRoboticsPublic/rosthrottle.git
+      version: 1.2.0
```

Comment:
```
ERROR: In file '''/home/travis/build/ros/rosdistro/melodic/distribution.yaml''': Could not validate source entry for repo rosthrottle-release with error [[[Could not validate repository with url https://github.com/UTNuclearRoboticsPublic/rosthrottle.git and version 1.2.0 from entry at line 6340. Error reason: Tags are not valid, but a tag 1.2.0 was found. Re: https://github.com/ros/rosdistro/pull/20286]]]
```

It looks like CI is failing because this line needs to refer to a branch, but it currently says `1.2.0` which is a tag. See [this section for more info](https://github.com/ros/rosdistro/blob/c076ccf4e9cfd3c6b2036568303fe5f1a4d49ca4/CONTRIBUTING.md#continuous-integration-indexing).